### DISCUSSION TITLE
How can I run a function after adonis5 startup

### DISCUSSION THREAD

**Discussion by bstsnail** [adonisjs/core#1551] [Help] [answered]
Hi, I'm new to adonis, I see the question [here](https://github.com/adonisjs/core/issues/544) , but I still don't know how to do it , can some one help me out? An example is better.

**Comment by thetutlage** [comment, marked as answer]
@bstsnail. You can do it using a preload file. 

- Create a new preload file using `node ace make:prldfile <name>`
- Make sure you select the `web` environment or all the environment. DO NOT LEAVE THE PROMPT SELECTION EMPTY.
- Next write `console.log` inside the newly created file to check it is getting loaded by AdonisJS automatically.
- If yes, then you can write any code inside it and it will be executed after the framework is ready

**Reply by bstsnail** [reply]
Great, it works.
But the step 2 is not showed as you mentioned, I select the 'During HTTP server' and it didn't work
![image](https://user-images.githubusercontent.com/16055826/91976111-0fb7e580-ed53-11ea-908a-df760e9652a7.png)
I hade to add 'web' in .adonisrc.json manually
![image](https://user-images.githubusercontent.com/16055826/91976193-337b2b80-ed53-11ea-81e1-d27952717db8.png)

**Reply by thetutlage** [reply]
You need to select using the `space` key. People usually forget to click space to select an option. I believe, we should improve the UX of this prompt

**Reply by bstsnail** [reply]
Get it , thanks a lot
